app/models/breed.rb

```diff
@@ -0,0 +1,7 @@
+class Breed < ApplicationRecord
+  serialize :images, Array
```

Comment:
This built-in ActiveRecord method was new to me, but I used it here so I could store an array as a string in the Rails database (which will not normally accept Hash or Array as a data type)